mne_bids/tests/test_dig.py

```diff
@@ -325,3 +325,25 @@ def test_convert_montage():
     assert pos['coord_frame'] == 'mri'
     assert_almost_equal(pos['ch_pos']['EEG 001'],
                         [-0.0313669, 0.0540269, 0.0949191])
+
+
+def test_electrodes_io(tmp_path):
+    """Ensure only electrodes end up in *_electrodes.json."""
+    raw = _load_raw()
+    raw.pick_types(eeg=True, stim=True)  # we don't need meg channels
+    bids_root = tmp_path / 'bids1'
+    bids_path = _bids_path.copy().update(root=bids_root, datatype='eeg')
+    (bids_root / 'sub-01' / 'ses-01' / 'eeg').mkdir(exist_ok=True)
+    write_raw_bids(raw, bids_path)
+
+    with open(bids_path.directory /
+              'sub-01_ses-01_acq-01_space-CapTrak_electrodes.tsv') as sidecar:
+        n_entries = len([line for line in sidecar
+                         if 'name' not in line])  # don't need the header
+        # only eeg chs w/ electrode pos should be written to electrodes.tsv
+        assert n_entries == len(
+            raw
+            .copy()
+            .pick_types(eeg=True)
+            .ch_names
+        )
```

Comment:
seems [my prior comment](https://github.com/mne-tools/mne-bids/pull/1023#discussion_r933557836) got lost: you should not need to do `raw.copy()` here. use `raw.get_channel_types()` and count the `'eeg'`s

Reply:
Nice trick, @drammock!

Reply:
awesome, thanks @drammock!